packages/fxa-auth-db-mysql/db-server/index.js

```diff
@@ -249,6 +249,18 @@ function createServer(db) {
   api.del('/account/:id/recoveryKey', withParams(db.deleteRecoveryKey))
   api.post('/account/:id/recoveryKey', withIdAndBody(db.createRecoveryKey))
 
+  api.get(
+    '/account/:id/subscriptionCapabilities/:clientId',
+    op(req => db.fetchSubscriptionCapabilities(req.params.id, req.params.clientId))
+  )
+  api.get('/account/:id/subscriptionCustomer', withIdAndBody(db.fetchSubscriptionId))
+  api.get('/account/:id/subscriptions', withIdAndBody(db.fetchSubscriptionItems))
+  api.del(
+    '/account/:id/subscriptions/:subscriptionId',
+    op(req => db.cancelSubscription(req.params.id, req.params.subscriptionId))
+  )
+  api.post('/account/:id/subscriptions', withIdAndBody(db.createSubscriptionItem));
```

Comment:
(I know this is still draft, but just calling out that we need to remember to update `docs/API.md` and `docs/DB_API.md` for this stuff before merging)

Reply:
Yeah, I'd done that on an earlier swing at this stuff. But then I totally reworked it all and figured I should get some eyes on the schema before I do that all over again 😅